src/robot/utils/robotpath.py

```diff
@@ -83,7 +83,7 @@ def abspath(path, case_normalize=False):
     3. Turn ``c:`` into ``c:\\`` on Windows instead of ``c:\\current\\path``.
     """
     path = normpath(path, case_normalize)
-    return normpath(_abspath(path), case_normalize)
+    return normpath(os.path.abspath(path), case_normalize)
```

Comment:
We still have need for `_abspath` due to IronPython 2.7.8 bug in `os.path.abspath`. This change thus needs to be reverted.